lib/money/money_parser.rb

```diff
@@ -1,31 +0,0 @@
-class MoneyParser
```

Comment:
some people are directly using this class `MoneyParser.parse(...)`. I don't think we should remove it

Reply:
If you're not doing a find/replace beforehand, Ruby will yell about a missing class. The interface stays the same so it's pretty harmless if you ask me?

Reply:
Instead of renaming the class to `DeprecatedMoneyParser` shouldn't we just add a deprecation warning when someone tries to use it? 
I'd be in favor of not adding that deprecation warning until we know what we'd like to use as an alternative.

Reply:
Fair enough, we can rename it later and leave it for now since this is not a full replacement.